capsules/extra/src/net/sixlowpan/sixlowpan_state.rs

```diff
@@ -955,7 +955,7 @@ impl<'a, A: time::Alarm<'a>, C: ContextStore> Sixlowpan<'a, A, C> {
                     }
                 }
             } else {
-                packet[0..payload_len].copy_from_slice(&payload[0..payload_len]);
+                return (None, Ok(()));
```

Comment:
Having peed at Google yesterday, I am reminded by their bathroom coding style flyer of a comment Johnathan has made on a few PRs. This would be slightly cleaner switched to a "guard" idiom, i.e. move

```rust
if !is_lowpan(payload) {
    return (None, OK());
}

// ... other code continues here not in an else block
```